pkgs/games/en-croissant/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, lib
+, dpkg
+, fetchurl
+, autoPatchelfHook
+, wrapGAppsHook
+, webkitgtk
+, libsoup
+, openssl_1_1
+}:
+
+stdenv.mkDerivation rec {
+  name = "en-croissant";
+  version = "0.7.1";
+
+  src = fetchurl {
```

Comment:
building from source is preferred approach in nixpkgs

https://github.com/franciscoBSalgueiro/en-croissant#building-from-source

Reply:
feel free to reach out if you run into issues